pkg/virt-api/rest/subresource.go

```diff
@@ -135,6 +135,30 @@ func (app *SubresourceAPIApp) putRequestHandler(request *restful.Request, respon
 	}
 }
 
+func (app *SubresourceAPIApp) getRequestHandler(request *restful.Request, response *restful.Response, validate validation, getURL URLResolver, v interface{}) {
```

Comment:
Any chance the name can be improved?
It does not "get" anything (sounds like a getter), and the name "Handler" does not really hint what it does exactly.

Sorry for not suggesting any other name here, I just do not really understand what this piece of code does eventually.

Reply:
Its a handler of HTTP GET request. I named it after similar handler for HTTP PUT: 

https://github.com/kubevirt/kubevirt/blob/f3c6c7df391b134d689da19b526e4c6f298c0170/pkg/virt-api/rest/subresource.go#L121

Maybe smth like `httpGetRequestHandler`?

Reply:
Yea, this suggested name is less confusing. Thanks!

Reply:
Updated to `httpGetRequestHandler`